src/coreclr/src/utilcode/util.cpp

```diff
@@ -2951,28 +2951,26 @@ LPWSTR *SegmentCommandLine(LPCWSTR lpCmdLine, DWORD *pNumArgs)
 //======================================================================
 // This function returns true, if it can determine that the instruction pointer
 // refers to a code address that belongs in the range of the given image.
-// <TODO>@TODO: Merge with IsIPInModule from vm\util.hpp</TODO>
-
-BOOL IsIPInModule(HMODULE_TGT hModule, PCODE ip)
+BOOL IsIPInModule(PTR_VOID pModule, PCODE ip)
```

Comment:
It would be nice to add comment about what pModule is, or maybe even give it a better name (e.g. `pModuleBaseAddress`)?